### PR TITLE
Document agent management flag

### PR DESCRIPTION
**Pull request by jcreixell**
#### PR Description

Documents the use and protocol of this experimental feature.

### CODE REVIEW

File: docs/sources/static/configuration/flags.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ Valid feature names are:
 * `integrations-next`: Enable [revamp]({{< relref "./integrations/integrations-next/" >}}) of the integrations subsystem
 * `dynamic-config`: Enable support for [dynamic configuration]({{< relref "./dynamic-config" >}})
 * `extra-scrape-metrics`: When enabled, additional time series  are exposed for each metrics instance scrape. See [Extra scrape metrics](https://prometheus.io/docs/prometheus/latest/feature_flags/#extra-scrape-metrics).
+* `extra-scrape-metrics`: When enabled, additional time series  are exposed for each metrics instance scrape. See [Extra scrape metrics](https://prometheus.io/docs/prometheus/latest/feature_flags/#extra-scrape-metrics).
```
Comment (marctc) [inline docs/sources/static/configuration/flags.md:36]: This line is duplicated

File: docs/sources/static/configuration/agent-management.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+base_config: |
+  <grafana_agent_config>
+# A set of snippets to be conditionally merged into the base configuration.
+snippets:
+  [ <snippet_name>: <snippet_content> ... ]
+```
+
+### grafana_agent_config
+
+This is a standard Grafana Agent [static mode configuration](https://grafana.com/docs/agent/latest/static/configuration/). Typically used to configure the server, remote_writes and other global configuration.
```
Comment (spartan0x117) [inline docs/sources/static/configuration/agent-management.md:70]:
```suggestion
This is a standard Grafana Agent [static mode configuration](https://grafana.com/docs/agent/latest/static/configuration/). Typically used to configure the server, remote_writes, and other global configuration.
```

**Review by spartan0x117 — APPROVED** [review]

File: docs/sources/static/configuration/agent-management.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
+---
+title: Agent Management
+weight: 700
+aliases:
+- ../../configuration/agent-management/
```
Comment (rfratto) [inline docs/sources/static/configuration/agent-management.md:5]: This is a new file, so you don't need to add the alias. You only need this if the file existed previously but got renamed or moved into a new folder.

File: docs/sources/static/configuration/agent-management.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        - targets: ['localhost:3100']
+      integration_configs:
+        node_exporter:
+          enabled: true
+    selector:
+      os: linux
+      app: app1
+```
+
+Note: both json and yaml responses are supported.
```
Comment (rfratto) [inline docs/sources/static/configuration/agent-management.md:123]:
```suggestion
Note: both JSON and YAML responses are supported.
```

What determines getting a JSON/YAML response? Accept/Content-Type header?

File: docs/sources/static/configuration/metrics-config.md [hunk 5]
```diff
@@ -331,6 +331,6 @@ remote_write:
 > **Note:** More information on the following types can be found on the Prometheus
 > website:
 >
-> * [`relabel_config`](https://prometheus.io/docs/prometheus/2.34/configuration/configuration/#relabel_config)
-> * [`scrape_config`](https://prometheus.io/docs/prometheus/2.34/configuration/configuration/#scrape_config)
-> * [`remote_write`](https://prometheus.io/docs/prometheus/2.34/configuration/configuration/#remote_write)
+> * [`relabel_config`](https://prometheus.io/docs/prometheus/2.37/configuration/configuration/#relabel_config)
```
[the remark this answers is not in the document]
Comment (rfratto) [reply docs/sources/static/configuration/metrics-config.md:334]: It should be the explicit version we actually embed to not mislead people about what fields are supported.

**Review by rfratto — COMMENTED** [review]
Overall LGTM, but you don't need my approval on this 👍

File: docs/sources/static/configuration/agent-management.md (hunk 4 above)
Comment (jcreixell) [reply docs/sources/static/configuration/agent-management.md:121]: I will add a note, but yes, via `Accept` header, defaulting to JSON

**Pull request merged**